hw/ip/spi_device/doc/dv/index.md

```diff
@@ -54,9 +54,10 @@ TL host interface into SPI Device.
 
 ### SPI Device agent
 [spi agent]({{< relref "hw/dv/sv/spi_agent/doc" >}}) is used to drive and monitor SPI items.
-Following special behavior is supported in spi_host_driver
-* Toggle clock when SPI is in idle state (csb=1)
-* During data transfer, there may be very long delay between each bit or byte of data
+There are 2 spi_agents in the testbench to minic a SPI host and a SPI device.
```

Comment:
minic? -> mimic?
Or maybe: to serve as a host and a device?

Reply:
Updated, thanks Cindy.